CubeScramble.py

```diff
@@ -4,20 +4,22 @@
 # Example: R, Ri or R2
 
 
-
-
-import random
-
+from random import choice
 
 # all possible moves on a Rubik's Cube
 
-list = ['U', 'Ui', 'U2', 'F', 'Fi', 'F2', 'R', 'Ri', 'R2', 'B', 'Bi', 'B2', \
+list = ['U', 'Ui', 'U2', 'F', 'Fi', 'F2', 'R', 'Ri', 'R2', 'B', 'Bi', 'B2',
     'L', 'Li', 'L2', 'D', 'Di', 'D2']
 
 # shuffle those moves
 
-random.shuffle(list)
 
-# print shuffled list
+def generate_random_moves(num_moves):
+    moves = []
+    for move in xrange(num_moves):
```

Comment:
Shit that `move` should have been `_`. I never use that iterator. If something is unused like this you name it `_`